### ISSUE TITLE
Add ability to generate Mirador config/state based on items added to widget

### ISSUE THREAD

**Issue by jkeck** [sul-dlss/vatican_exhibits#92]
We need to be able to generate an initial mirador configuration based on the items that were added by the curator via the widget UI.  The config/state should be capable of initializing a mirador viewer with all of the items available and arranged in panes according the layout rules below.

From sul-dlss/exhibits#1002

> * 1 item -> layout obvious...
> * 2 items -> typical 2-up, side-by-side layout
> * 3 items -> 2-up, side-by-side layout for top row, 3rd item takes full-width of bottom row
> * 4 items -> 2 up in top row, 2 up in bottom

**Comment by mejackreed** [comment]
Row/Column configs:
```javascript
"windowObjects": [{
  loadedManifest: "http://iiif.harvardartmuseums.org/manifests/object/299843",
  viewType: "ImageView",
  slotAddress: "row1.column1"
  },{
  loadedManifest: "http://media.nga.gov/public/manifests/nga_highlights.json",
  slotAddress: "row1.column2",
  viewType: "ImageView"
}],
```

**Comment by jkeck** [comment]
@mejackreed (or @aeschylus) do you know if the 3-item layout described is possible?

It's not via the Mirador UI, but figured we may be able to force it in a custom derived configuration.

**Comment by mejackreed** [comment]
Yeah I can't seem to figure out the 3 item layout config option.. Kicking the can on it and just going w/ 4

**Comment by mejackreed** [comment]
Resolved in #98
